main/depend.common

```diff
@@ -701,6 +701,7 @@ module_radiation_driver.o: \
                 module_ra_rrtmg_lwk.o \
                 module_ra_rrtmg_swk.o \
 		module_ra_cam.o \
+		module_ra_farms.o \
```

Comment:
@liujake 
Jake,
There is a separate PR for this mod, which is a more general fix than just for DA.
Please remove it.

Reply:
I thought this should disappear after you merge your PR. Apparently not.

Reply:
No such luck.
I can easily remove this line via github, if you like.

Reply:
Removed it online.